src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -777,7 +781,7 @@ private static <T> T makeCopy(final T original) {
     /**
      * Returns a node name for a new node that doesn't clash with any we've
      * currently got.
-     * 
+     *
```

Comment:
Unnecessary change - please undo. See the [contributing guide's coding style section](https://github.com/jenkinsci/docker-plugin/blob/master/CONTRIBUTING.md#submitting-pull-requests) for details.